Controller/MediaAdminController.php

```diff
@@ -109,6 +117,132 @@ public function listAction(Request $request = null)
     }
 
     /**
+     * @param Request $request
+     *
+     * @throws \Symfony\Component\Security\Core\Exception\AccessDeniedException|\LogicException
+     *
+     * @return \Symfony\Bundle\FrameworkBundle\Controller\Response|\Symfony\Component\HttpFoundation\Response
+     */
+    final public function multiUploadAction(Request $request)
+    {
+        $this->admin->checkAccess('create');
+
+        $providerName = $this->getRequest()->get('provider');
```

Comment:
`$providerName` needs to be checked against a whitelist of valid provider service ids. I shouldn't be able to make the site crash because I use `logger` as a provider.